src/test/java/seedu/address/logic/parser/IVolunteerParserTest.java

```diff
@@ -50,19 +51,28 @@ public void parseCommand_clear() throws Exception {
 
     @Test
     public void parseCommand_delete() throws Exception {
```

Comment:
perhaps u can rename this method to `parseCommand_volunteerDelete`  for clarity sake?

Reply:
gud idea